lib/pry/helpers/completer_helpers.rb

```diff
@@ -0,0 +1,64 @@
+class Pry
+  module Helpers
+    module CompleterHelpers
```

Comment:
This is mostly because this methods are static. I don't like the idea to have them inside instance methods. But due to https://github.com/pry/pry/pull/1583#issuecomment-260186419, I've moved them out of `class << self`.